.goreleaser.yml

```diff
@@ -20,6 +20,9 @@ builds:
     - darwin
   goarch:
     - amd64
+  hooks:
+    pre: sed -i s/~mailchain-version~/{{.Version}}/ ./cmd/mailchain/internal/http/handlers/swagger.go
```

Comment:
Also tried to export this to a script and calling the script here, but I was getting an error without any details of the error. Interestingly, running an empty shell script here also threw an error.